documentation/src/main/asciidoc/elasticsearch-integration.asciidoc

```diff
@@ -31,10 +31,10 @@ There could be various reasons to prefer this over an "embedded Lucene" approach
 * benefit from Elasticsearch's out of the box horizontal scalability features
 * explore the data updated by an Hibernate powered application using the Elasticsearch dashboard integrations such as Kibana
 
-There are a couple drawbacks compared to the embedded Lucene approach though:
+There are a couple of drawbacks compared to the embedded Lucene approach though:
 
 * incur a performance penalty of remote RPCs both for index updates and to run queries
-* need to buy and manage additional servers
+* need to manage an additional service
```

Comment:
Well, you can run your Es service on the same VM/bare metal server. Feel free to ignore this part but I thought it was a bit weird.